docs/configure-artifact-repository.md

```diff
@@ -75,7 +75,8 @@ $ cat > policy.json <<EOF
          "Effect":"Allow",
          "Action":[
             "s3:PutObject",
-            "s3:GetObject"
+            "s3:GetObject",
+            "s3:ListBucket"
```

Comment:
Would existing workflows break if this permission is not there when s3 is used? If so, we also need to document this in upgrading guide.

Reply:
wil do!

Reply:
(and yes)